### PR TITLE
firestore test

### LINKED ISSUE #30: upload art backend
# hook frontend uploading of art to data-base.
- be sure that art is added in with the account info that uploaded it.

### PR DESCRIPTION
**Pull request by snowmang1**
- new: firestore hooked up and working, Initialized in main. used in overview page
- fix: urls now stored to db, from form field
- resolves #30, fix: db will manage art urls with named keys
- fix: commented code for clarity
- fixed: firestore_test branch merge with current version of main

### CODE REVIEW

**Comment by snowmang1** [comment]
do not merge anything with main until this has been merged so that all the backend does not create lethal conflicts again.

**Review by dhelms1 — APPROVED** [review]

**Pull request merged**
